aws-sagemaker-hosted-scorer-cpp/README.md

```diff
@@ -0,0 +1,93 @@
+# DAI Deployment Template for Sagemaker Hosted C++ Scorer
+
+## Overview
+
+### Build Image
+
+Run the following command to build the docker image.
+
+```bash
+docker build -t <aws_account_id>.dkr.ecr.<region>.amazonaws.com/h2oai/sagemaker-hosted-scorer:<tag> .
```

Comment:
Can we get this as part of official gradle build. That image get pushed into our Vorvan registry?